LAUNCHABLE-CREATION-GUIDE.md

```diff
@@ -0,0 +1,149 @@
+
+## Runtime Configuration
+
+Determine your runtime environment:
+
+### VM Mode (USUALLY THE MOST OPTIMAL, SIMPLE CONFIGURATION)
+
+Provides a GPU VM in the cloud, with Docker, Python, and CUDA preinstalled. Ubuntu 22.04 as the base OS.
+
+- Does not allow you to choose a container but you have the ability to install dependencies based upon your use case within a terminal/notebook session
+- If you have a container that requires a private registry or API keys to pull the image, select the VM Mode option. Then, in your uploaded notebook file, you should specify the steps to authenticate and pull the image.
+
+### With Container(s)
+
+If you require container(s) running for the code sample, you can also specify a Docker Image / Docker Compose YAML.
+
+This adds significant time to the provisioning process, since it takes time to build images/containers. Expect anywhere between 5-15 minutes of additional time.
+
+#### Option 1: Featured Containers
+
+Best for simple demos with a quick setup with a Jupyter notebook experience. These are pre-developed Docker images created by the Brev team. They consist of popular images used commonly by Brev users and come with JupyterLab pre-installed.
+
+#### Option 2: Docker Compose YAML (Advanced)
+
+Best for showcasing complex workflows with multiple containers.
+
+If you have your own Docker image(s) on a public registry, you can manually add containers to be provisioned on the instance. If you already have a pre-created [Docker Compose](https://docs.docker.com/compose/) YAML file, you can also upload it!
+
+**Note:** If your container requires a private registry or API keys to pull the image, click the back button below and select the VM Mode option. Then, in your uploaded notebook file, you should specify the steps to authenticate and pull the image.
+
+## JupyterLab/Tunnels Configuration
+
+Choose whether you want [JupyterLab](https://jupyter.org) on your instance. This depends on if you want your users to go through a Jupyter Notebook/IDE for interacting with the Launchable (HIGHLY RECOMMENDED). Declining means a user will need to SSH into the GPU.
+
+You can also configure whether you want to pre-expose any tunnels or services or set firewall rules on the VM when it is provisioned. This is helpful for any networking necessary for the Launchable workload and your Launchable's users won't need to mess with any networking.
+
+Examples of use cases for tunnels include:
+- [Gradio](https://gradio.app)/[Streamlit](https://streamlit.io) UIs
+- Microservices/NIMs
+- Communication with Servers
+- And more!
+
+## Compute Configuration
+
+[Select an NVIDIA GPU](https://brev.nvidia.com/environment/new/public) for your use case, scroll to select your configuration. Please note the following, and choose the most reasonable minimum-cost configuration for your needs:
```

Comment:
don't know if it really matters but this ends up redirecting me to: https://brev.nvidia.com/environment/new (maybe just cuz I'm admin?)